docs/content/singleton-actors.md

```diff
@@ -0,0 +1,79 @@
+# Singleton Actors
+
+Singleton Actors allow a single source of truth for both operations and data across a multi-server simulation. They are server-side authoritative [Actors](https://docs.unrealengine.com/en-us/Programming/UnrealArchitecture/Actors) that are restricted to one instantiation on SpatialOS. Singleton Actors allow a single source of truth for both operations and data across a multi-server simulation. There are two kinds of Singleton Actors:
+
+* **Public Singleton Actors** - Singleton Actors which are replicated to clients. `GameState` is a Public Singleton Actor.
+* **Private Singletons** - Singleton Actors which are not accessible to clients. `GameMode` is a Private Singleton Actor.
```

Comment:
I still don't like this public/private thing. But we can change it when we actually implement.

Reply:
Don't like the terms or splitting this up into two groups?

Reply:
Terms. Private to indicate server-side only feels a bit off. But for now it's fine.